Gruntfile.js

```diff
@@ -71,7 +71,7 @@ module.exports = function (grunt) {
             },
             "node-mac": {
                 "dest"      : "<%= downloads %>",
-                "src"       : "http://nodejs.org/dist/v<%= node.version %>/node-v<%= node.version %>-darwin-x86.tar.gz"
+                "src"       : "http://nodejs.org/dist/v<%= node.version %>/node-v<%= node.version %>-darwin-x64.tar.gz"
```

Comment:
We might want to leave the mac node to be 32 bit, as our native node extensions are still 32 bit and they would not load on 64 bit node.